__tests__/api.test.js

```diff
@@ -22,20 +24,167 @@ describe('GET /api/topics', () => {
+        .expect(200)
+    })
+    test('get /api/articles request responds with an array of articles objects with a comment_count property and date in decending order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(res => {
+            expect(res.body.articles).toBeInstanceOf(Array);
+            expect(res.body.articles).not.toBeEmpty();
+            for (let i = 0; i < res.body.articles.length; i++) {
+                expect(res.body.articles[i]).toBeInstanceOf(Object);
+                expect(res.body.articles[i]).not.toBeEmpty();
+                expect(res.body.articles[i]).toHaveProperty('article_id');
+                expect(res.body.articles[i]).toHaveProperty('title');
+                expect(res.body.articles[i]).toHaveProperty('body');
+                expect(res.body.articles[i]).toHaveProperty('votes');
+                expect(res.body.articles[i]).toHaveProperty('topic');
+                expect(res.body.articles[i]).toHaveProperty('author');
+                expect(res.body.articles[i]).toHaveProperty('created_at');
+                expect(res.body.articles[i]).toHaveProperty('comment_count');
+            }
+        })
+    })
+    test('get /api/articles date property is in descending order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(res => {
+            expect(res.body.articles).toBeSortedBy('created_at', { descending: true })
+        })
+    })
+    });
 
+describe('GET /api/articles/:article_id', () => {
+    test('responds with a 200 status code', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+    })
+    test('get /api/articles/:article_id request responds with an article object with the correct properties', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(res => {
+            
+            for (let i = 0; i < res.body.article.length; i++) {
+                expect(res.body.article).toBeInstanceOf(Object);
+                expect(res.body.article).not.toBeEmpty();
+                expect(res.body.article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),})
+            }
+        })
+    })
+})
 
+describe('GET /api/articles/:article_id', () => {
+    test('responds with a 200 status code', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+    })
+    test('get /api/articles/:article_id request responds with an array of comments objects with the correct properties', () => {
+        return request(app)
```

Comment:
I would also add a test for if given a valid ID but had no comments that it returns an empty comments array (200)